include/triton/Dialect/TritonGPU/IR/LinearLayoutConversions.h

```diff
@@ -282,6 +282,11 @@ LinearLayout chooseScaledMfmaScaleLayout(
     MLIRContext *ctx, int dotOperandIdx,
     const std::vector<std::vector<int32_t>> &dotOperandWarpBasis,
     ArrayRef<int64_t> dotOperandShape, unsigned mfmaMDim);
-} // namespace mlir::triton::gpu
 
+// Create a LinearLayuot similar to mfmaLayout where each thread holds 8
+// consecutive elements. The goal is to store dwordx4 in the epilogue.
+LinearLayout chooseMfma8Layout(AMDMfmaEncodingAttr mfmaLayout,
```

Comment:
I think the name can be confusing--we don't have a mfma8 layout per se. Maybe something like `chooseMfmaLikeStoreLayout` would be better?